app/test/spec/zeebe-api/zeebe-api-spec.js

```diff
@@ -2544,18 +2424,22 @@ function mockZeebeNode(options = {}) {
     ...(options.log || {})
   };
 
-  const ZeebeNode = {
-    ZBClient: options.ZBClient || function() {
-      return {
+  class CamundaClient {
```

Comment:
As indicated above, let's call this MockCamunda8`, or rename `Camunda8` to `CamundaClient`.

Reply:
@abdul99ahad Your commit did not get it quite right, see https://github.com/camunda/camunda-modeler/pull/4817/commits/e20f4bb76c3c0a0bf9fd84090e0aa0f7aac48058 for how to declare class types:

```
typeof Camunda8 -> class/constructor <Camunda8>
Camunda8 -> instance of the type <Camunda8>
```